conf/cmi/block.block.chatleijuke.yml

```diff
@@ -7,17 +7,19 @@ dependencies:
     - system
   theme:
     - hdbt_subtheme
-id: genesyschatsuunte
+id: chatleijuke
 theme: hdbt_subtheme
 region: attachments
 weight: 0
 provider: null
-plugin: genesys_suunte
+plugin: chat_leijuke
 settings:
-  id: genesys_suunte
-  label: 'Genesys Chat (SUUNTE)'
+  id: chat_leijuke
+  label: 'Chat Leijuke SUUNTE'
   label_display: '0'
   provider: helfi_platform_config
+  chat_title: 'Hammashoidon chat'
```

Comment:
I assume we need proper English translation for this and FI/SV translations as well.